### PR TITLE
Target core module compilation to Java 1.6

### PR DESCRIPTION
**Pull request by qtdyn**

### CODE REVIEW

**Comment by Col-E** [comment]
I assume the project is being downgraded to 6 so that it can be used directly in a 6 classpath?

File: core/src/test/java/software/coley/versionpatcher/CoreTests.java [hunk 1]
```diff
@@ -1,44 +1,64 @@
...
+				public boolean accept(File dir, String name) {
+					return name.matches("Java\\d+\\w+.class");
+				}
+			});
+		} catch (Exception ex) {
+			throw new IllegalStateException(ex);
+		}
+	}
+
+	private final File targetFile;
```
Comment (Col-E) [inline core/src/test/java/software/coley/versionpatcher/CoreTests.java]: Generally prefer fields to all be define at the top.

**Comment by qtdyn** [comment]
> I assume the project is being downgraded to 6 so that it can be used directly in a 6 classpath?

Yup exactly

**Comment by xtrm-en** [comment]
:pensive:

**Pull request closed without merging**